wscript

```diff
@@ -561,7 +561,7 @@ video_output_features = [
     } , {
         'name': '--wayland-scanner',
         'desc': 'wayland-scanner',
-        'func': check_program('wayland-scanner', 'WAYSCAN')
+        'func': check_program('wayland-scanner', '>= 1.15.0')
```

Comment:
WAYSCAN is actually the variable into which the binary path gets put.

This is not a pkg-config check :) .
```python
def __wayland_scanner_cmd__(ctx, mode, dir, src, vendored_file):
    return "${{WAYSCAN}} {0} < {1} > ${{TGT}}".format(
        mode,
        "${SRC}" if vendored_file else "{}/{}".format(dir, src)
    )
```
No idea why it didn't just throw an exception at us for this in this function because of WAYSCAN not being defined...